kong-config.yaml

```
@@ -40,5 +40,16 @@ services:
           add:
             headers:
             - X-Frame-Options:ALLOWALL
+  - name: authenticid
+    url: https://idva-kong-egress-${ENVIRONMENT_NAME}.apps.internal:61443
```

Ahh don't we need `/authenticid` as the path here?
```suggestion
    url: https://idva-kong-egress-${ENVIRONMENT_NAME}.apps.internal:61443/authenticid
```

---

strip_path: false keeps the path component on the request